components/Redirect.vue

```diff
@@ -0,0 +1,12 @@
+<template></template>
+
+<script>
```

Comment:
The component is nothing doing right now. I suggest to move all logic in the component and remove mixin.